FrontEndReact/src/View/Admin/Add/ImportTasks/AdminImportAssessmentTasks.js

```diff
@@ -74,32 +81,82 @@ class AdminImportAssessmentTask extends Component {
                     />
                 }
                 { errorMessage &&
                     <ErrorMessage
-                        add={this.props.addAssessmentTask}
+                        add={addAssessmentTask}
                         resource={"Assessment Task"}
                         errorMessage={errorMessage}
                     />
                 }
                 { validMessage!=="" &&
                     <ErrorMessage
-                        add={this.props.addAssessmentTask}
+                        add={addAssessmentTask}
                         error={validMessage}
                     />
                 }
-                <div id="outside">
+                <Box className="card-spacing">
+                    <Box className="form-position">
+                        <Box className="card-style">
+                            <Box className='form-spacing'>
+                            <Typography id="importAssessmentTasksTitle" sx={{mb: 3}} variant="h5"> Import Assessment Tasks </Typography>
+                                <Box className="form-input">
+                                    <Box sx={{mb: 3}}>
+                                <Box>
+                                    Please select the course you would like to import assesments tasks from.
+                                </Box>
+                    </Box>
+                    <Box sx={{ mb: 3}}>
+                            <Box>
+                                <CourseDropdown
+                                    id="courseSelected"
+                                    setSelectedCourse={this.setSelectedCourse}
+                                />
+                            </Box>
+                       
+                    </Box>
+                            <Box sx={{display:"flex", justifyContent:"flex-end", alignItems:"center", gap: "20px"}}>
+                                <Button 
+                                onClick={() => {
+                                    confirmCreateResource("AssessmentTask")
+                                }}
+                                id="" className="">   
+                                    Cancel
+                                </Button>
+
+                                <Button 
+                                onClick={() => {
+                                    confirmCreateResource('AssessmentTask');
+                                }}
+                                id="importAssessmentTasks" className="primary-color"
+                                variant="contained"
+                                >   
+                                    Import Tasks
+                                </Button>
+                                </Box>
+                            </Box>
+                            </Box>
+                        </Box>
+                    </Box>
+                </Box>
+                    
+                {/* <div id="outside">
```

Comment:
Any reason to keep this commented out code?

Reply:
The commented out code is no longer needed! I am going to go ahead and delete it!